Mathlib/RingTheory/Localization/Algebra.lean

```diff
@@ -142,3 +144,21 @@ lemma AlgHom.toKerIsLocalization_isLocalizedModule (f : A →ₐ[R] B) :
     (algebraMapSubmonoid_map_eq M f)
 
 end Algebra
+
+namespace Polynomial
+
+/-- If `S` is the localization of `R` at a submonoid `M`, then `S[X]` is the localization of
+`R[X]` at `M.map Polynomial.C`.
```

Comment:
Somehow the doc string seems to be confused, shouldn't this say:
```suggestion
/-- If `A` is the localization of `R` at a submonoid `S`, then `A[X]` is the localization of
`R[X]` at `S.map Polynomial.C`.
```

Reply:
Do you mean that docstrings are not invariant under alpha-equivalence? 😱